```java
18:49:02,750 [ERROR] -> [Help 1]
--
  |   |   | org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.14.0:testCompile (default-testCompile) on project apache-james-mailbox-store: Compilation failure
  |   |   | /home/jenkins/jenkins-agent/workspace/james_ApacheJames_PR-3032/mailbox/store/src/test/java/org/apache/james/mailbox/store/search/mime/FieldImpl.java:[27,8] org.apache.james.mailbox.store.search.mime.FieldImpl is not abstract and does not override abstract method getSafeRaw() in org.apache.james.mime4j.stream.Field
```

--- a/mailbox/store/src/test/java/org/apache/james/mailbox/store/search/mime/FieldImpl.java
+++ b/mailbox/store/src/test/java/org/apache/james/mailbox/store/search/mime/FieldImpl.java
@@ -49,6 +49,11 @@ public class FieldImpl implements Field {
     }
 
     @Override
+    public ByteSequence getSafeRaw() {
+        return null;
+    }
+
+    @Override
     public int hashCode() {
         return Objects.hash(name, body);
     }

